src/components/Toggle/Toggle.js

```diff
@@ -0,0 +1,25 @@
+import React, {Component} from 'react';
+import './Toggle.css';
+
+export class Toggle extends Component {
+
+    constructor(props) {
+        super(props);
+    }
+
+    render() {
+
+        const {type = 'checkbox', isToggleChecked, onChange} = this.props;
+
+        return (
+            <label className="toggle-wrap">
+                <div>{this.props.label}</div>
```

Comment:
Please destruct `label ` from  this.props in code above

Reply:
Done